src/argus_htmx/templates/htmx/incidents/_incident_filterbox.html

```diff
@@ -1,13 +1,13 @@
 {% load widget_tweaks %}
-<form>
-    <fieldset
-            hx-get="{% url 'htmx:incident-list' %}"
-            hx-include="this"
-            hx-trigger="change delay:100ms"
-            hx-target="#table"
-            hx-swap="outerHTML"
-            hx-push-url="true"
-    >
+<form id="incident-filter-box"
+      hx-get="{% url 'htmx:incident-list' %}"
+      hx-include="#table-refresh-info, #incident-filter-box"
+      hx-trigger="keydown[keyCode==13], change delay:100ms"
+      hx-target="#table"
+      hx-swap="outerHTML"
+      hx-push-url="true"
+      onkeydown="if (event.keyCode === 13) event.preventDefault();">
+    <fieldset>
```

Comment:
You change to post on enter, not after delay? What's the `onkeydown` for?

Reply:
So it used to be only `change delay` for htmx. But hitting enter while one of the form fields were focused also caused a get request to be fired. This was not an ajax request, and not triggered by htmx. I'm not quite sure what the origin of the request is. However, that request would only add the filterbox form fields to the query parameters, but we also need to include the current page size. So I had to prevent the keydown enter event, and add it as and hx-trigger. hx-trigger now has both triggers

If you can figure out a better solution, please feel free :D